README.md

```diff
@@ -103,108 +103,115 @@ To learn more about Istio, please refer to the
 
-[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fgke-istio-gce-demo&page=editor&tutorial=README.md)
+### Deploying Demo from a Local Machine
 
-All the tools for the demo are installed. When using Cloud Shell execute the following
-command in order to setup gcloud cli.
+_NOTE: If the demo is being deployed via Cloud Shell, this section can be skipped._
 
-```console
-gcloud init
+For deployments from a local machine, you will need to have access to a [bash](https://www.gnu.org/software/bash/) shell with the following tools installed:
+
+* [Google Cloud SDK (v204.0.0 or later)](https://cloud.google.com/sdk/downloads)
+* [kubectl (v1.10.0 or later)](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [git](https://git-scm.com/)
+
+Use `git` to clone this project to your local machine:
+
+```shell
+git clone --recursive https://github.com/GoogleCloudPlatform/gke-istio-gce-demo
 ```
 
-### Supported Operating Systems
+Note that the `--recursive` argument is required to download dependencies provided via a git submodule.
 
-This project will run on macOS, Linux, or in a [Google Cloud Shell](https://cloud.google.com/shell/docs/).
+When downloading is complete, change your current working directory to the new project:
 
-### Tools
+```shell
+cd gke-istio-gce-demo
+```
 
-When not using Cloud Shell, the following tools are required.
+Continue with the instructions below running all commands from this directory.
 
-In order to use the code in this demo you will need to have have access to a
-bash shell with the following tools installed:
+### Deploying Demo from Google Cloud Shell
 
-* A [GCP project](https://cloud.google.com/) with billing enabled
-* [Google Cloud SDK (v204.0.0 or later)](https://cloud.google.com/sdk/downloads)\
-* [kubectl (v1.10.0 or later)](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* git
-* A bash-compatible shell
+_NOTE: This section can be skipped if the cloud deployment is being performed from a local computer and the instructions from the previous section were followed._
+
+Click the button below to open the demo in your [Google Cloud Shell](https://cloud.google.com/shell/docs/).
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fgke-istio-gce-demo&page=editor&tutorial=README.md)
+
+All the tools necessary for the demo are already installed in Cloud Shell. To prepare [gcloud](https://cloud.google.com/sdk/gcloud/) for use in Cloud Shell, execute the following command in the terminal at the bottom of the browser window you just opened:
+
+```console
+gcloud init
+```
+
+Respond to the prompts and continue with the following deployment instructions.
```

Comment:
I believe it selects a default and it may not matter what they choose since the code will be deployed to the location of the cluster, which the user sets in the properties file before running the demo setup. I've added some clarifying text, however.